packages/ui/theme-builder/app/stores/appearance-store.ts

```diff
@@ -0,0 +1,32 @@
+import { create } from 'zustand';
+
+interface AppearanceStoreState {
+  appearance: 'light' | 'dark';
+  direction: 'ltr' | 'rtl';
+  devMode: 'on' | 'off';
+  animations: 'on' | 'off';
+}
+
+interface AppearanceStoreActions {
+  setAppearance: (appearance: 'light' | 'dark') => void;
+  setDirection: (direction: 'ltr' | 'rtl') => void;
+  setDevMode: (devMode: 'on' | 'off') => void;
+  setAnimations: (animations: 'on' | 'off') => void;
+  reset: () => void;
+}
+
+const initialState: AppearanceStoreState = {
+  appearance: 'light',
+  direction: 'ltr',
+  devMode: 'on',
+  animations: 'on',
+};
+
+export const useAppearanceStore = create<AppearanceStoreState & AppearanceStoreActions>(set => ({
```

Comment:
Nice one on adding Zustand here, it's such a simple data store and it removes a ton of prop drilling ❤️